peps/pep-0783.rst

```diff
@@ -0,0 +1,187 @@
+PEP: 783
+Title: Emscripten Packaging
+Author: Hood Chatham <roberthoodchatham at gmail.com>
+Sponsor: Łukasz Langa <lukasz at python.org>
+Discussions-To:
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 28-Mar-2025
+Post-History: `18-Mar-2025 <https://discuss.python.org/t/84996>`__
+
+Abstract
+========
+
+This PEP proposes a new platform tag series ``pyodide`` for binary Python package
+distributions for the Pyodide Python runtime.
+
+`Emscripten <https://emscripten.org/>`__ is a complete open source compiler
+toolchain. It compiles C/C++ code into WebAssembly/JavaScript executables, for
+use in JavaScript runtimes, including browsers and Node.js. The Rust language
+also maintains an Emscripten target.
```

Comment:
Should this reference PEP 776 as the specification for the platform being packaged?

Reply:
Perhaps use the _Requires_ header?

Reply:
That may be desirable as well for metadata purposes - but I was thinking more of an in-text reference.